### DISCUSSION TITLE
What are Groups?

### DISCUSSION THREAD

**Discussion by dave1quovadimus** [williamtroup/Calendar.js#91] [Q&A]
I see how to get, clear and remove groups, but what are they? Why would I use them?

**Comment by sottey** [comment]
In Things3, they have the concept of "Areas". I use it for "Home" and " Work". Groups seem to be the same concept. Very useful, especially when combined with Search and the ability to toggle visibility by group.

**Comment by AntreasPetsas** [comment]
Is it possible to have each group in different column on a specific day? To be more precise, let's assume I have 6 events and two groups. Let's name them "vehicle 1" and "vehicle 2". So what I need is even if the times of the events are not overlapping, i want to have the day splitted equal to the number of groups. So for the example, i will have the Day splitted into two sections and each section will have its 3 events. This should be scalable. E.g if i have 4 groups, the day should be splitted into 4 sections. As you can guess, this could only be available on "Day Specific" View

**Reply by AntreasPetsas** [reply]
is there any "hack-around-work" that this could be done? 
At the moment when two events are overlapping you are splitting that timeslot into two sections.

**Reply by AntreasPetsas** [reply]
Happy coding then! :)

**Comment by toomanylogins** [comment]
Can somebody please clarify my understanding of groups/locations and types in timeline view. Using a groups and locations I could achieve the equivalent of a restaurant = group by location, however dont see how if I wanted to display say outside/inside then the tables which are outside or inside ? Or similarly if we had teams = group  and people = types ? There is no mechanism for a hierarchical display or have I misunderstood.
Thanks
